packages/core/src/WebPlayer.ts

```diff
@@ -147,6 +147,41 @@ export default class WebPlayer extends EventEmitter {
     }
   }
 
+  setVolume({ 
+    change, 
+    percentage 
+  }: { 
```

Comment:
From a programming logic perspective I would have seen something more simplified. In the line of this

```
let newVolume = change ? this.tech.volume + change : percentage / 100;
if (newVolume > 1) {
  newVolume = 1;
}
if (newVolume < 0) {
  newVolume = 0;
}
this.tech.volume = newVolume;
this.tech.volume > 0 ? this.isMuted && this.unmute() : this.mute();
```